### ISSUE TITLE
Staging Review finding: Entry points/static page(s) not updated to point to new tool 

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#86820]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** VYE Team
**Product:** Verify Your Enrollment
**Feature:** Verify Your Enrollment

## Findings details
**VA.gov Experience Standard - issue:** [User can't determine where a page is within the VA.gov website.](https://design.va.gov/about/experience-standards/#user-can-determine-where-a-page-is-within-the-vagov-website)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA)

### Description

CAIA's [IA spec identifies several entry points](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/mgib-verify-enrollment.md#nav) for this form. None have been updated on staging to reflect this new product.

## Recommended action

Update links from entry point pages to go to the new application instead of WAVE.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** User flow

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by mnorthuis** [comment]
@jason-gcio To remove this launch blocker, the entry point on the [tool static landing page](https://www.va.gov/education/verify-school-enrollment/) needs to be implemented.  This is the sign in widget that your team is working on with CAIA.   Once that is done, this issue can be closed. 

The other entry points listed in the[ IA spec](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/mgib-verify-enrollment.md#nav) (numbers 2&3) will be added by CAIA soon, but are not required for launch.  And the 4th entry point listed - the about form page - is going to be removed per a stakeholder conversation.

**Comment by jsimonVA** [comment]
@mnorthuis worked with @RLHecht  and confirmed that the ReactWidget is working and this has lifted our launch blocker for the [tool static landing page](https://www.va.gov/education/verify-school-enrollment/)

**Comment by mnorthuis** [comment]
Excellent!  You can close this issue at your discretion.
